### PR TITLE
Fixed bullet points not having full stops at the end

### LINKED ISSUE #3997: Grammar in education/apply for a student developer pack
### What article on docs.github.com is affected?

[Education/apply for a student developer pack](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/apply-for-a-student-developer-pack).

### What part(s) of the article would you like to see updated?

Requirement for student pack bullet points. 

### Additional information

Small grammar errors but the bullet points do not end with full stops like other bullet points in the docs and look out of place. 
Opening a pull request now to fix this.

### PR DESCRIPTION
**Pull request by fluteds**
### Why:

**Closes #3997**

### What's being changed:

Added full stops to the end of the bullet points explaining requirements. 

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@fluteds Thanks so much for opening an issue and a PR! I'll get this triaged for review ⚡

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/apply-for-a-student-developer-pack.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ versions:
...
 To be eligible for the {% data variables.product.prodname_student_pack %}, you must:
-- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution
-- Have a verifiable school-issued email address or upload documents that prove your current student status
+- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution.
+- Have a verifiable school-issued email address or upload documents that prove your current student status.
 - Have a [{% data variables.product.prodname_dotcom %} user account](/articles/signing-up-for-a-new-github-account)
-- Be at least 13 years old
+- Be at least 13 years old.
 
 Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.
```
Comment (mtgsquad) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/apply-for-a-student-developer-pack.md:23]: Seems Good, Lets Do This!

**Review by mtgsquad — APPROVED** [review]
seems good, i think everything is fixed most issues are solved i think

**Comment by ethanpalm** [comment]
👋  Hi @fluteds, thanks for opening a PR! This isn't something we're looking to change right now, so I'm closing this PR. Thanks for your passion and interest for improving the documentation!

Take a look at the [contributing.md](https://github.com/github/docs/blob/main/CONTRIBUTING.md/) to learn how to contribute in this repo. You can open a [discussion](https://github.com/github/docs/discussions) if you have any questions or check out issues with the [help wanted](https://github.com/github/docs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label. :sparkles:

**Pull request closed without merging**